modules/inflowwind/src/IfW_FlowField.f90

```diff
@@ -107,13 +110,26 @@ subroutine IfW_FlowField_GetVelAcc(FF, IStart, Time, PositionXYZ, VelocityUVW, A
 
    ! Copy positions or transform based on wind box rotation
    if (FF%RotateWindBox) then
-      do i = 1, NumPoints
-         Position(:, i) = GetPrimePosition(PositionXYZ(:, i))
-      end do
-   else
+      if (present(PosOffset)) then
+         do i = 1, NumPoints
+            Position(:, i) = GetPrimePosition(PositionXYZ(:, i) + PosOffset)
+         end do
+      else
+         do i = 1, NumPoints
+            Position(:, i) = GetPrimePosition(PositionXYZ(:, i))
+         end do
+      end if
+   else if (present(PosOffset)) then
+      Position = PositionXYZ + spread(PosOffset, 2, NumPoints)
+   else 
       Position = PositionXYZ
    end if
 
+   ! If
```

Comment:
?

Reply:
The if statement above it was rearranged a couple of times, must have missed deleting part of it that was commented out